config/locales/en.yml

```diff
@@ -1242,10 +1242,11 @@ en:
           <span>Make sure to <a href="https://docs.openshift.com/container-platform/latest/dev_guide/routes.html#creating-routes">add the correct route</a></span>
         endpoint_apicast_2_html: >
           Public address of your API gateway in the %{environment_name} environment.
-        api_backend_https: "Private address of your API that will be called by the API gateway."
+        api_backend_https: |
+          Private address of your API that will be called by the API gateway.
+          <span class="protocol-warn">For end-to-end encryption your private base URL scheme should use a secure protocol.</span>
         api_backend_http: |
           Private address of your API that will be called by the API gateway.
```

Comment:
@thomasmaas @hallelujah 
Before rendering the form  (https://github.com/3scale/porta/blob/master/app/views/api/integrations/apicast/configuration_driven/_configuration.html.slim#L27) we are checking if the protocol is 'http' or 'https', but not if 'ws' or 'wss' (https://github.com/3scale/porta/blob/master/app/helpers/api/integrations_helper.rb#L141).

After submitting the form with wrong protocol we get this message:
![schemes](https://user-images.githubusercontent.com/13486237/73437585-e5312480-434c-11ea-8341-f916b4ce37eb.png)

I think the hint messages are obsolete now, we should rephrase them, WDYT?

Reply:
@damianpm we are working for on prem issue, meaning apiap is enabled, meaning this url is provided at backend edit. I don't understand why it's obsolete?

<img width="1792" alt="Screenshot 2020-01-30 11 30 09" src="https://user-images.githubusercontent.com/54224/73441825-e9147500-4353-11ea-9de5-98cc9e8c55e2.png">

Reply:
I'm talking when No-APIAP in API > Integration > configuration
![error](https://user-images.githubusercontent.com/13486237/73443228-95eff180-4356-11ea-8562-40b83caa876e.png)